website/docs/guides/guides-upgrade.md

```diff
@@ -4,39 +4,39 @@ title: Upgrade to a newer version
 sidebar_position: 1
 ---
 
-## Foreword
-
-> Please note that this guide is work in progress!
-> If there is one step you should follow one by one, then it is the backup of the current system.
-> If you find any issues with the guide, then please tell us so (by [opening a new issue on Github](https://github.com/lansuite/lansuite/issues/new))!
-
 ## Prerequisites
 
-* A system having both Git and Composer up and running if you want to use the Git Master
-* A current file and database backup
-* Your target system running PHP >= 7.0 and MySQL >= 5.6.3
+* Your target system need to run PHP >= 8.1 and MySQL >= 5.7
 * A fair bit of time
 
 ## Preparation
 
-### Plan
+### Download an official release
 
-It is recommended to tell your users (and fellow administrators) in advance that your LanSuite instance won't be 100% functional available during the upgrade.
```

Comment:
As per comment:
The guide lists all steps that should be done to avoid issues before/during/after the upgrade.
And lack of communication / planning is one very often seen.

So I would leave it in.
I'm confident that readers can figure out for themselves if they need to do that or not

Reply:
I personally think we need to set boundaries here between us and the user/operator. We can't fix everything for them.

I never have seen such a note in any other system / open source upgrade path. I do think a short and concise upgrade doc that covers the requirements is read more often than a "this covers everything" doc.

Anyway. I would still remove it and focus on the system. But if you think this is useful, we can keep it in.

Reply:
Fixed in https://github.com/lansuite/lansuite/pull/794/commits/4ab9c4d50b3ab8258c032c78c6dfbe534c70442e

Reply:
It's the [debian way of doing things](https://www.debian.org/releases/stable/i386/release-notes/ch-upgrading.en.html#inform-users) ;)